packages/block-editor/src/components/link-control/style.scss

```
@@ -3,6 +3,12 @@ $block-editor-link-control-number-of-actions: 1;
 .block-editor-link-control {
 	position: relative;
 	min-width: $modal-min-width;
+
+	.components-popover__content & {
+		min-width: unset;
```

`unset` is not supported in Internet Explorer.

https://caniuse.com/#feat=css-unset-value

Given that `auto` is the initial value for `min-width`, I expect that could work:

```suggestion
		min-width: auto;
```